docs/notification-routing/disable-oomkill-notifications.rst

```diff
@@ -0,0 +1,11 @@
+Disable "OOMKill" Notifications
+===================================
+
+Configure Robusta to not send OOMKill notifications by disabling the built-in OOMKill playbook. 
+
+.. code-block:: yaml
+
+    disabledPlaybooks:
+    - PodOOMKill
+
+Similarly you can to disable any built-in notification using the name of the playbook. Find all the built-in playbooks `here <https://github.com/robusta-dev/robusta/blob/cc686ed893d24aed1096fe805d483dfd7007508a/helm/robusta/values.yaml#L169>`_ and `here <https://github.com/robusta-dev/robusta/blob/cc686ed893d24aed1096fe805d483dfd7007508a/helm/robusta/values.yaml#L113>`_
```

Comment:
You're linking to a specific revision of the file here, which means that someone who clicks the link will always see an older version of the file, even after it changes. Please link to the files on master. (LMK if you need help with it.)